ui/packages/consul-ui/app/components/consul/upstream-instance/list/index.hbs

```diff
@@ -63,6 +63,16 @@ as |mode|}}
               {{item.LocalBindSocketPath}}
             </dd>
           </dl>
+          <dl class="local-bind-mode">
+            <dt>
+              <span>
+                Local bind mode
+              </span>
+            </dt>
+            <dd>
+              <span>mode:</span> {{mode.value}}
```

Comment:
Maybe check in with design, but I think we generally uppercase the first letter of field/property names

Reply:
According to the designers we follow a pattern of lowercase field names. Ex. Exposed Paths
<img width="300" alt="Screen Shot 2021-07-16 at 9 54 37 AM" src="https://user-images.githubusercontent.com/19161242/125959116-18e6226d-bf7f-4105-aa67-105cf6d1404e.png">

Following the same pattern here. No changes made.